concepts/80 Troubleshooting/45 Component Content is not Scrollable/25 Scrollbar is Displayed but Component is Overlapped.md

```diff
@@ -0,0 +1,3 @@
+Sometimes a component displays a scrollbar but elements below the component overlap its content. For example, your task is to place a component between a header and footer with fixed heights but this issue occurs.
```

Comment:
Once again - the edits might depend on whether we are talking about overlapping the scrollbar or overlapping component content...